### PR TITLE
Adds NFData instances for all types

### PR DESCRIPTION
**Pull request by ssbothwell**
In reference to issue [#52](https://github.com/andrewthad/chronos/issues/52) this adds NFData instances for all Chronos types.

### CODE REVIEW

File: chronos.cabal [hunk 1]
```diff
@@ -57,10 +57,11 @@ library
     -- it is OPTIONS_HADDOCK-hidden
       Chronos.Internal.CTimespec
   build-depends:
-    , aeson >= 1.1 && < 1.5
+      aeson >= 1.1 && < 1.5
```
Comment (andrewthad) [inline chronos.cabal:60]: Can you put these leading commas back? They help me sort dependencies lists in my text editor.

Comment (ssbothwell) [reply chronos.cabal:60]: oh sorry about that. My editor did some funny reformatting on the cabal file and i missed that part.

**Comment by chessai** [comment]
You are missing some bang patterns 

rnf _ = ()

should be

rnf !_ = ()

**Comment by ssbothwell** [comment]
Oops! I'll fixed that right now.

**Comment by andrewthad** [comment]
One last request: bump the version to 1.1.2 and then add a new changelog entry mentioning the `NFData` instances with the release date as `2020.??.??`.

**Comment by ssbothwell** [comment]
Version updated!

**Pull request merged**